Examples/package.lisp

```diff
@@ -1,4 +1,4 @@
 
 (defpackage #:clim-demo
   (:use #:clim-extensions #:clim #:clim-lisp)
-  (:export #:demodemo #:run-drawing-tests #:tabledemo))
+  (:export #:demodemo #:run-drawing-tests #:tabledemo #:misc-tests))
```

Comment:
Idea of quick start function is fine but the exported symbol list keeps growing. How about we export `demodemo` and a single function which takes a `test-name` argument and runs the test directly?

Reply:
Fixed (added single function run-demo which is exported from clim-demo package).